tugas kelompok 7

humaid-nurudin opened the pull request:
hasil kerja kelompok 7 dari anggota bernama humaid dengan nim a11.2019.12373 dari kelp a11.4617

humaid-nurudin committed:
tugas kelompok 7

hasil kerja kelompok 7 dari anggota bernama humaid dengan nim a11.2019.12373 dari kelp a11.4617

andikajuliant commented:
by Andika Juliant (A11.2019.12266)

andikajuliant commented:
[TUGAS_KELOMPOK_7 _PPL.docx](https://github.com/angelitaanap/PPL_A11.2019.12282/files/8825846/TUGAS_KELOMPOK_7._PPL.docx)
by Andika Juliant A11.2019.12266

humaid-nurudin committed:
tugas kelompok 7 yang dilengkapi

tugas kelompok 7 yang dikerjakan oleh salah satu anggotanya bernama humaid yang berisi studi kasus yang telah dilengkapi.

andikajuliant commented:
[TUGAS_KELOMPOK_7_A11.2019.12266.docx](https://github.com/angelitaanap/PPL_A11.2019.12282/files/8834032/TUGAS_KELOMPOK_7_A11.2019.12266.docx)
by Andika Juliant #1

humaid-nurudin committed:
tugas kelompok 7 cost

menambahkan cost budgeting dan cost controlling pada tugas kelompok.
